src/languages/lsl.js

```diff
@@ -0,0 +1,133 @@
+/*
+    Language: Linden Scripting Language
+    Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+    Author: Builder's Brewery <buildersbrewery@gmail.com>
+    Category: common, scripting
```

Comment:
Also, please remove the indentation within the meta comment, it's not supposed to be there and may suddenly stop working if anyone decides to rewrite its parsing for any reason.